editor/gui/output_dock.cpp

```diff
@@ -45,7 +64,10 @@ void OutputDock::draw(float deltaMilliseconds)
 					ImGui::PushStyleColor(ImGuiCol_Text, (const ImVec4&)EditorSystem::GetSingleton()->getNormalColor());
 				}
 
-				ImGui::Text("%s", outputString.second.c_str());
+				if (((filter & (int)OutputState::General) && !(outputString.first == "Warning" || outputString.first == "Error")) || ((filter & (int)OutputState::Warning) && (outputString.first == "Warning")) || ((filter & (int)OutputState::Error) && (outputString.first == "Error")))
```

Comment:
break super long statement into multiple lines
```suggestion
				if ( ((filter & (int)OutputState::General) && !(outputString.first == "Warning" || outputString.first == "Error")) ||
				((filter & (int)OutputState::Warning) && (outputString.first == "Warning")) ||
				((filter & (int)OutputState::Error) && (outputString.first == "Error")) )
```

Reply:
I tried it but it is failing the build on Github since it doesn't follow the clang-format. If I format it, then it forces it to be in 1 line.